doc/userguide/rules/payload-keywords.rst

```diff
@@ -691,11 +693,25 @@ the reassembled stream.
 The checksums will be recalculated by Suricata and changed after the
 replace keyword is being used.
 
+.. _pkt-data:
+
+pkt_data
+--------
+
+``pkt_data`` is a sticky buffer that allows for payload inspection from the
+beginning of the normalized packet data. As a sticky buffer, ``pkt_data``
```

Comment:
what is a normalized packet data?

Reply:
That's post-processing? I admit that I went for a long shot, using explanation from a different source. Should I remove this entirely? For us, does it make more sense to say post-reassembly, or not even that?